indico/modules/events/abstracts/controllers/management.py

```diff
@@ -228,7 +257,7 @@ def _process(self):
             return redirect(self.list_generator.get_list_url())
         list_kwargs = self.list_generator.get_list_kwargs()
         return WPManageAbstracts.render_template('management/abstract_list.html', self._conf, event=self.event_new,
-                                                 **list_kwargs)
+                                                 actions=AbstractAction, **list_kwargs)
```

Comment:
not needed after https://github.com/indico/indico/pull/2656/commits/3f76f44956daa52e443e644e8653af847734cba6#r84282337